functions/assignOfflineContact.ts

```diff
@@ -0,0 +1,89 @@
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import {
+  responseWithCors,
+  bindResolve,
+  error400,
+  error500,
+  success,
+} from '@tech-matters/serverless-helpers';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+type EnvVars = {
+  TWILIO_WORKSPACE_SID: string;
+  TWILIO_CHAT_TRANSFER_WORKFLOW_SID: string;
+};
+
+export type Body = {
+  targetSid?: string;
+  finalTaskAttributes: string;
+};
+
+export const handler: ServerlessFunctionSignature = TokenValidator(
+  async (context: Context<EnvVars>, event: Body, callback: ServerlessCallback) => {
+    const client = context.getTwilioClient();
+
+    const response = responseWithCors();
+    const resolve = bindResolve(callback)(response);
+
+    const { targetSid, finalTaskAttributes } = event;
+
+    try {
+      if (targetSid === undefined) {
```

Comment:
Not big deal, but you can use: 
`if (!targetSid) { ... } `

Reply:
There are cases where we want to be able to send empty strings, and checking for `if (!targetSid) { ... }` won't allow it. That's why, in general, in all the Serverless functions you will see me checking like this at first, then if the empty string (or whatever) is not valid, that error can be addressed in more detail. This just checks that the parameter at least exists.